f2/src/pdf/HowDidItStartView.js

```diff
@@ -0,0 +1,83 @@
+/** @jsx jsx */
+import { jsx } from '@emotion/core'
+import { containsData } from '../utils/containsData'
+import { testdata } from '../ConfirmationSummary'
+import { Text, View, Image } from '@react-pdf/renderer'
+import { pdfStyles } from './pdfStyles'
+import { formatList } from '../utils/formatList'
+import { DescriptionItemView } from './DescriptionItemView'
+import line from '../images/line.png'
+
+export const HowDidItStartView = (props) => {
+  const lang = props.lang
+
+  const summary = []
+  let overviewLine = ' '
+
+  const howdiditstart = {
+    ...testdata.formData.howdiditstart,
+    ...props.data.formData.howdiditstart,
+  }
+
+  if (howdiditstart.howDidTheyReachYou.length > 0) {
+    //Obtain all the array data into the summary array
+    howdiditstart.howDidTheyReachYou.map((key) =>
+      summary.push(
+        key === 'howDidTheyReachYou.others' && howdiditstart.others !== ''
+          ? howdiditstart.others
+          : key.toLowerCase(),
+      ),
+    )
+
+    overviewLine =
+      lang['confirmationPage.howDidItStart.overviewPrefix'] +
+      formatList(summary, {
+        pair: lang['default.pair'],
+        middle: lang['default.middle'],
+        end: lang['default.end'],
+      })
+  }
+
+  return (
+    <View style={pdfStyles.section}>
+      <Text style={pdfStyles.title}>
+        {lang['confirmationPage.howDidItStart.title']}
+      </Text>
+      {containsData(howdiditstart) ? (
+        <View>
+          <Text style={pdfStyles.sectionContent}>{overviewLine}</Text>
+          <DescriptionItemView
+            title="confirmationPage.howDidItStart.email"
+            description={howdiditstart.email}
+            lang={lang}
```

Comment:
See this everywhere `lang={lang}`.  Why are we manually managing translation? This is what Lingui is for.

Reply:
It seems react-pdf doesn't work with any hook functions. I spent some time and decided to work around by passing in an parameter.  If you figure it out, definitely let me know. Thanks!

Reply:
If react-pdf doesn't work with react hooks and our translation framework requires hooks, why did we use this library?

Reply:
I have done my research and I liked pdfkit, but I found that react pdf is based on pdfkit and easier to work with React. And this library is very popular. That's why I picked this library. And this is the best library working with React I can find. 
That doesn't mean react pdf doesn't work with React hooks, that could be just I missed something, if you could help, that would be great.